dart/lib/src/sentry_attachment/sentry_attachment.dart

```diff
@@ -88,6 +93,17 @@ class SentryAttachment {
             contentType: 'image/png',
             attachmentType: SentryAttachment.typeAttachmentDefault);
 
+  SentryAttachment.fromViewHierarchy(SentryViewHierarchy sentryViewHierarchy)
+      : this.fromLoader(
+          loader: () => Uint8List.fromList(
+              utf8JsonEncoder.convert(sentryViewHierarchy.toJson())),
+          filename: 'view-hierarchy.json',
+          contentType: 'application/json',
+        );
+  // TODO: rollback this change, relay is not accepting this type
+  // if sentt with a single item in the envelope
+  // attachmentType: SentryAttachment.typeViewHierarchy);
```

Comment:
Is this comment still up to date? Will the constructor be removed in the future?

Reply:
Good catch, forgot to remove it